cypress/e2e/notes-100/access-shared-note.cy.js

```diff
@@ -1,16 +1,39 @@
 import {waitForModel, homepageSetup} from '../../support/utils'
 
-
-describe('select-a-note', () => {
-  context('Open the link with a shared note', () => {
+// From https://github.com/bldrs-ai/Share/issues/1072
+describe('Notes 100 - Access shared note', () => {
+  context('Access shared link containing the path to the general notes', () => {
     beforeEach(() => {
```

Comment:
now this one should just be the visit and wait:
```
beforeEach(() => {
  cy.visit('/share/v/p/index.ifc#c:-133.022,131.828,161.85,-38.078,22.64,-2.314;i:')
  waitForModel()
})
```

Reply:
Ah ok, makes sense to split into two different methods.